runtimes/common/src/constants.rs

```diff
@@ -363,14 +363,19 @@ pub mod governance {
 	}
 }
 
+pub mod timestamp {
+	use super::*;
+
+	#[allow(clippy::integer_division)]
+	pub const MINIMUM_PERIOD: u64 = SLOT_DURATION / 2;
+}
+
 pub mod multisig {
 	use super::*;
 
-	parameter_types! {
-		pub const MaxSignitors: u32 = 64;
-		pub const DepositBase: Balance = DEPOSIT_STORAGE_ITEM;
-		pub const DepositFactor: Balance = DEPOSIT_STORAGE_BYTE;
-	}
+	pub const MAX_SIGNITORS: u32 = 64;
+	pub const DEPOSIT_BASE: Balance = DEPOSIT_STORAGE_ITEM;
+	pub const DEPOSIT_FACTOR: Balance = DEPOSIT_STORAGE_BYTE;
```

Comment:
Is there a reason to drop the parameter_types macro and using the Const?

Reply:
As I said above, this is more a matter of a personal preference. It uses the Rust primitives types instead of a macro. I probably should not have done this refactoring in this PR as well, so I will probably revert these changes and migrate everything to the new syntax in a different PR, if you agree this is better.

Reply:
I reverted the changes in here: https://github.com/KILTprotocol/kilt-node/pull/797/commits/08e3236e09db6ea4974bb3729551d4ef1f106993. I will change all of them if you agree in a separate PR.